euth/projects/overwrites.py

```diff
@@ -0,0 +1,14 @@
+from django.utils.translation import ugettext_lazy as _
+
+from adhocracy4.projects.enums import Access
+
+
+def overwrite_access_enum_label():
+    Access.__labels__ = {
+        Access.PRIVATE: _('Only invited users can see content and can '
+                          'participate (private).'),
+        Access.PUBLIC: _('All users can see content and can participate '
+                         '(public).'),
+        Access.SEMIPUBLIC: _('All users can see content, only invited users '
+                             'can participate (semi-public).')
```

Comment:
Maybe we can put that into a4? We tried to overwrite the enum with two options in there, but as that is not working anyway...
I started something last week, but then didn't finish because I just wanted the update to get working and then forgot about that. https://github.com/liqd/adhocracy4/pull/713 Maybe you can use it?!